apps/ff-site/src/app/_styles/globals.css

```diff
@@ -266,4 +266,15 @@
   .copy-to-clipboard-button {
     @apply hover:text-brand-400;
   }
+
+  /* SHARE ARTICLE */
+  .share-article {
+    &.title {
+      @apply text-brand-300;
+    }
+
+    &.tooltip {
+      @apply hover:text-brand-400;
+    }
+  }
```

Comment:
Does this actually work as expected? I don't think this targets `share-article-title` and `share-article-tooltip` but `share-article title` and `share-article tooltip`? Right? 😵‍💫🙈

Reply:
Sorry, forget what I said! I just realized it's `&.`

The only change I would make is:

```
  .share-article {
    &.share-article-title {
      @apply text-brand-300;
    }

    &.share-article-tooltip {
      @apply hover:text-brand-400;
    }
  }
```

I think it's what we are trying to do in the last couple of PRs - need to refactor the rest later! 🙏🏼

Reply:
Though if we don't actually have a `share-article` class - it can be removed.